performance/venv.py

```diff
@@ -240,4 +240,4 @@ def create_virtualenv(python):
 def exec_in_virtualenv(options):
     venv_python = create_virtualenv(options.python)
     args = [venv_python, "-m", "performance"] + sys.argv[1:] + ["--inside-venv"]
-    os.execv(args[0], args)
+    run_cmd(args)
```

Comment:
Hum. Can you please keep os.execv() for UNIX/BSD, but use run_cmd() for other platforms?

And please also add sys.exit(0) after run_cmd().